cmd/scollector/collectors/collectors.go

```diff
@@ -325,3 +325,17 @@ func skipMetric(index string) bool {
 	}
 	return true
 }
+
+type tsIntegrator func(int64, float64) float64
+
+func getTsIntegrator() tsIntegrator {
+	var i float64
```

Comment:
these variable names are not descriptive enough to indicate what is going on here.